css/main.css

```diff
@@ -240,3 +241,11 @@ footer {
   background-color: #1f1d2b;
   color: #fff;
 }
+
+.main-content {
+  display: none;
+}
+
+.display-page {
+  display: block;
+}
```

Comment:
- [ ] In both stylelintrc.json and eslitnrc.json you seem to be adding more rules than those stated in this [link](https://github.com/microverseinc/linters-config/tree/master/javascript), kindly ensure you use the exact rules as they are shared by other students and keeps the project professional.